action.yaml

```diff
@@ -19,6 +19,8 @@ inputs:
   deployHostUrl:
     description: 'Location of the Armory Cloud API which will handle your deployment'
     default: 'api.cloud.armory.io'
+  extraArgs:
```

Comment:
Do we want to leave this generic and call it extra args or do we want to say "name" and try to compose the command. This is easier but is, in my opinion, a less ideal user experience.

Reply:
Yes, but we will need an entry-point.sh, I was being lazy, I will update PR.

Reply:
- [ ] make this nicer ;P